.golangci.yaml

```diff
@@ -51,10 +55,23 @@ linters-settings:
   goimports:
     local-prefixes: "github.com/openfga/openfga"
   importas:
-    # Do not allow unaliased imports of aliased packages.
-    no-unaliased: true
-    # Do not allow non-required aliases.
-    no-extra-aliases: false
+    no-unaliased: true # Do not allow unaliased imports of aliased packages.
+    no-extra-aliases: false # Do not allow non-required aliases.
     alias:
       - pkg: github.com/openfga/api/proto/openfga/v1
         alias: openfgav1
+
+issues:
+  exclude-use-default: false
```

Comment:
By not having this configuration set within the linter config, we were ignoring certain issues. For example the `errcheck` linter issues.